### PR TITLE
[14.0] [FIX] fix url redirection

### PR DESCRIPTION
**Pull request by sebastienbeau**
Url redirection is broken https://github.com/shopinvader/odoo-shopinvader/issues/1026

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@simahawk @PierrickBrun @lmignon

File: shopinvader/tests/common.py [hunk 1]
```diff
@@ -146,6 +146,9 @@ def setUpClass(cls):
             context=dict(cls.env.context, tracking_disable=cls.tracking_disable)
         )
         CommonMixin._setup_backend(cls)
+        # It seem that setUpComponent / setUpRegistry loose stuff from
```
Comment (simahawk) [inline shopinvader/tests/common.py:150]: how?

**Review by simahawk — COMMENTED** [review]
should it be backported to v13?

File: shopinvader/tests/common.py (hunk 1 above)
Comment (sebastienbeau) [reply shopinvader/tests/common.py:150]: no idea ! I just tested it.
You can try to read a computed field (like the url_key) before the setup and after.
Before you have the value after it's not here anymore (crazy thing).

Comment (sebastienbeau) [reply shopinvader/tests/common.py:150]: If I flush before then all data are kept.
I will try to investigate later, but if you have the time, help are welcome

Comment (simahawk) [reply shopinvader/tests/common.py:150]: but then wouldn't be better to call `self.env["base"].flush()` and add TODO for later?

Comment (sebastienbeau) [reply shopinvader/tests/common.py:150]: yes better ! Done

**Comment by simahawk** [comment]
need a rebase as https://github.com/shopinvader/odoo-shopinvader/pull/1016 got merge

**Comment by sebastienbeau** [comment]
> should it be backported to v13?

Maybe, we should test if we have the issue in v13 (as the orm have a lot of change during this last version) and then try to backport

**Comment by sebastienbeau** [comment]
now it should be ok

**Review by simahawk — APPROVED** [review]

**Pull request merged**